packages/fast-components-react-base/src/media/Media.props.ts

```diff
@@ -0,0 +1,48 @@
+import * as React from "react";
+
+/**
+ * Media HTML tags
+ */
+export enum MediaHTMLTags {
+    img = "img",
+    picture = "picture"
+}
+
+export interface IMediaProps {
+    /**
+     * The HTML alt attribute value is important for overall accessibility, providing a textual 
+     * alternative to non-text content 
+     */
+    alt: string;
+
+    /**
+     * The HTML itemScope attribute defines the scope of associated metadata and is used for associating the 
+     * element context with definitions on schema.org (important for seach engine optimization)
+     */
+    itemScope?: boolean;
+
+    /**
+     * The round image option
+     */
+    round?: boolean;
+
+    /**
+     * The HTML src attribute for the image element (specifying src will chose the image element rather than the picture element)
+     */
+    src?: string;
+
+    /**
```

Comment:
Are these different (srcSet and pictureSrcSet)? In your implementation, it seems like we should be able to pass a single prop.

Reply:
We could pass one string but the standard <img /> srcset passes in a pixel value suffixed with "w" for width. 

<img 
   srcset="quilt_3/large.jpg  1240w, 
           quilt_3/medium.jpg  620w,
           quilt_3/small.jpg   310w"
   sizes="(min-width: 41.25em) 38.75em,
          calc(100vw - 2.5em)"
   src="quilt_3/medium.jpg"
   alt="A crazy quilt whose irregular fabric scraps are fit into a lattice of diamonds." />

It can also handle things like ratios:
<img src="standard.jpg" srcset="retina.jpg 2x, super-retina.jpg 3x" />

I think keeping them separate removes any confusion about what element will be rendered. Here is more about using srcSets:
http://alistapart.com/article/responsive-images-in-practice

Reply:
We could pass one string but the standard <img /> srcset passes in a pixel value suffixed with "w" for width. 

```
<img 
   srcset="quilt_3/large.jpg  1240w, 
           quilt_3/medium.jpg  620w,
           quilt_3/small.jpg   310w"
   sizes="(min-width: 41.25em) 38.75em,
          calc(100vw - 2.5em)"
   src="quilt_3/medium.jpg"
   alt="A crazy quilt whose irregular fabric scraps are fit into a lattice of diamonds." />
```

It can also handle things like ratios:
`"<img src="standard.jpg" srcset="retina.jpg 2x, super-retina.jpg 3x" />"
`
I think keeping them separate removes any confusion about what element will be rendered. Here is more about using srcSets:
http://alistapart.com/article/responsive-images-in-practice

Reply:
Sure - I think though that if the aspect ratio is different, per Mat's feedback - we would want to use the picture element. If it's simply a size multiplier, that's likely different. Either way, if these are kept seperate I think we would potentially want different definitions rather than having these combined together. IE, what would happen if I passed information into both srcset props?

Reply:
I mean, I will say that it makes sense to apply the same rules to a passed `srcset` in `picture` as `srcset` on a stand-alone `img`. _Technically_ you could have multiple `source` elements for art direction, with `srcset`/`sizes` attributes on each for the sake of arguably-over-optimizing the requests.